### PR TITLE
updated pagination ref and patterns table

### PR DESCRIPTION
**Pull request by OlgaPodo**
Updated link for pagination to REST API guidelines  and added Alternate Key to the pattern table

### CODE REVIEW

File: graph/GuidelinesGraph.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Guidelines](https://github.com/microsoft/api-guidelines/blob/master/Guidelines.m
 |----------------------------------------------------------------------------------------------------|
 | :heavy_check_mark: **MUST** support \$select on resource to enable properties projection |
 | :ballot_box_with_check: **SHOULD** support \$filter with eq, ne operations on properties of entities for collections| 
-| :heavy_check_mark: **MUST** support [server-side pagination](http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_ServerDrivenPaging) using a [nextLink](http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ControlInformationnextLinkodatanextL) for collections |
+| :heavy_check_mark: **MUST** support [server-side pagination](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#981-server-driven-paging) using a [nextLink](http://docs.oasis-open.org/odata/odata-json-format/v4.01/odata-json-format-v4.01.html#sec_ControlInformationnextLinkodatanextL) for collections |
 | :ballot_box_with_check: **SHOULD** support pagination $top, $skip and $count for collections |
```
Comment (dkershaw10) [inline graph/GuidelinesGraph.md]: This line also doesn't make sense.
Maybe:  
"**SHOULD** support pagination of collections using $top and $skip (or $skipToken)"
"**SHOULD** support $count for collections"

File: graph/GuidelinesGraph.md [hunk 2]
```diff
@@ -417,6 +417,7 @@ Recommended API Design patterns:
 | Evolvable Enums     | The ability to extend enumerated types without breaking changes.       | [Evolvable Enums](./patterns/evolvable-enums.md) |
 | Change Tracking     | The ability to keep API consumers in sync with changes without polling.       | [Change Tracking](./patterns/change-tracking.md) |
 | Namespace     | The ability to organize resource definitions into a logical set.     | [Namespace](./patterns/namespace.md) |
+| Alternate Key     | The ability to uniquely identify and query recources using alternate key.     | [Alternate Key](./patterns/alternate-key.md) |
```
Comment (dkershaw10) [inline graph/GuidelinesGraph.md]: recources -> resources

using alternate key -> using an alternate key

**Review by dkershaw10 — CHANGES_REQUESTED** [review]
Address the comments and I'll approve

File: graph/GuidelinesGraph.md (hunk 1 above)
Comment (OlgaPodo) [reply graph/GuidelinesGraph.md]: Agree. also added a link to the client-driven pagination and changed server-side to server -driven for consistency

File: graph/GuidelinesGraph.md [hunk 3]
```diff
@@ -417,6 +418,7 @@ Recommended API Design patterns:
 | Evolvable Enums     | The ability to extend enumerated types without breaking changes.       | [Evolvable Enums](./patterns/evolvable-enums.md) |
 | Change Tracking     | The ability to keep API consumers in sync with changes without polling.       | [Change Tracking](./patterns/change-tracking.md) |
 | Namespace     | The ability to organize resource definitions into a logical set.     | [Namespace](./patterns/namespace.md) |
+| Alternate Key     | The ability to uniquely identify and query recources using an alternate key.     | [Alternate Key](./patterns/alternate-key.md) |
```
Comment (dkershaw10) [inline graph/GuidelinesGraph.md]: recources -> resources

**Review by dkershaw10 — APPROVED** [review]
Approved - but you should fix the typo...

File: graph/GuidelinesGraph.md (hunk 3 above)
Comment (OlgaPodo) [reply graph/GuidelinesGraph.md]: fixed

**Pull request merged**